pydicom/dataelem.py

```diff
@@ -433,13 +433,21 @@ def value(self) -> Any:
     @value.setter
     def value(self, val: Any) -> None:
         """Convert (if necessary) and set the value of the element."""
+        # Ignore backslash characters in these str or byte VRs
+        #   Based on Part 5, Section 6.2 and
+        exclusions = [
+            'LT', 'OB', 'OD', 'OF', 'OL', 'OV', 'OW', 'ST', 'UN', 'UT',
+            'OB/OW', 'OW/OB', 'OB or OW', 'OW or OB',
+        ]
```

Comment:
I'm not sure why some of the VRs are there (like FL/FD/AT, etc) since they'll fail the `isinstance(val, (str, bytes))` check

Reply:
Also the `'US' in self.VR`, condition which should be `int`.

Reply:
Hm, is it possible to assign a `bytes` value to a numeric VR like `US`? In this case it would make sense to check for all numeric VRs, in case a byte happens to be the representation of `/`.